### PR TITLE
 drop use of lib2to3 and EOL python2.7

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
it looks like you'll need to migrate to travis-ci.com https://app.travis-ci.com/github/suds-community/suds/pull_requests

**Comment by graingert** [comment]
see https://github.com/pypa/setuptools/issues/2086

**Comment by graingert** [comment]
@phillbaker would you consider moving suds-community into https://github.com/jazzband/ ?

**Comment by phillbaker** [comment]
Thanks for your work on this @graingert, looks like this will close https://github.com/suds-community/suds/issues/57.

> it looks like you'll need to migrate to travis-ci.com

I've actually been meaning to migrate to github actions, i'll get that going.

> would you consider moving suds-community into https://github.com/jazzband/ ?

I would consider this - can you open a separate issue for that so it can be tracked?

**Review by phillbaker — COMMENTED** [review]
I've swapped over to github actions on master, can you rebase this branch please?

Can you also remove the reference to supporting 2.7 in the README? https://github.com/suds-community/suds/blob/77dbccdb5d9e54c0fb36b1bc629793cc01d9c792/README.md#L100

File: setup.py [hunk 1]
```diff
@@ -574,27 +573,6 @@ def _test_cmd_string(self):
...
-    else:
-        from distutils.command.build_py import build_py_2to3
-        distutils_cmdclass["build_py"] = build_py_2to3
-
-    # Teach Python's urllib lib2to3 fixer that the old urllib2.__version__ data
-    # member is now stored in the urllib.request module.
-    import lib2to3.fixes.fix_urllib
-    for x in lib2to3.fixes.fix_urllib.MAPPING["urllib2"]:
-        if x[0] == "urllib.request":
-            x[1].append("__version__")
```
[the remark this answers is not in the document]
Comment (graingert) [reply setup.py:594]: @phillbaker I did apply this patch to 2to3 before I ran it

**Comment by graingert** [comment]
@phillbaker can you approve CI on this PR?

**Comment by phillbaker** [comment]
Thanks for the work here, going to bump the major version and release it.

**Comment by graingert** [comment]
can you also release a wheel?

```python
pip install -U pip setuptools wheel
pip install -U build twine
build --sdist --wheel
twine upload dist/*
```

**Comment by phillbaker** [comment]
Let me know if you have any issues, I released it as a beta to start: https://pypi.org/project/suds-community/1.0.0b1/#files

**Pull request merged**